Makefile

```diff
@@ -189,6 +193,26 @@ push-ruby-build-image: ## Push the Ruby build-time container image to a registry
 push-ruby-image: ## Push the Ruby test runtime container image to a registry.
 	docker push $(RUN_IMAGE_PREFIX)ruby:$(TEST_INFRA_VERSION)
 
+##@ Build PSM related container images
+
+all-psm-images: envoy-image xds-server-image ## Build all psm related container images to a registry.
```

Comment:
Nit: All of these targets that don't correspond to a file on the filesystem should probably be [`PHONY`](https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html#:~:text=A%20phony%20target%20is%20one,name%2C%20and%20to%20improve%20performance.)

Reply:
Thank you so much to let me know, I didn't know this before. :)